src/super_gradients/recipes/dataset_params/supervisely_persons_dataset_params.yaml

```diff
@@ -1,5 +1,8 @@
+dataset_dir: /data/supervisely-persons
+batch_size: 8
+
 train_dataset_params:
-  root_dir: /data/supervisely-persons
+  root_dir: ${..dataset_dir}
```

Comment:
I see you are using ${..dataset_dir} instead of ${dataset_params.dataset_dir} which is the syntax we usually use.
To be honest, I wonder why we are not using this local syntax which seems to be more robust. Do you have any downside of using ${..dataset_dir}  in mind?

Reply:
not that I know of :)